src/services/notes.tsx

```diff
@@ -28,4 +28,10 @@ const editNote = (day: string, index: number, newNote: string) => {
   store.set(day, dayArr);
 };
 
-export default { getDay, addNote, deleteNote, editNote };
+const archiveNotes = (week: string) => {
+  const archiveStore = new Store({ name: week });
+  archiveStore.store = store.store;
+  store.store = {};
+};
```

Comment:
I like it! Do these archived stores all just get saved into the Wenda AppData folder? If so, perhaps there's a way to store them in /Wenda/archive to reduce eventual clutter? Worth looking into.

Reply:
See suggested change - found directory option in [docs](https://github.com/sindresorhus/electron-store#cwd).

Reply:
Good idea.